src/plugin-slots/EditFileErrorAlertsSlot/README.md

```diff
@@ -0,0 +1,46 @@
+# File and Videos Page Alerts Slot
+
+### Slot ID: `org.openedx.frontend.authoring.edit_file_error_alerts.v1`
+
+## Description
+
+This slot is used to add alerts to the course file edit page.
+
+## Example
+
+### Additional Alert on Files Page
+![🍞 in breadcrumbs slot](./screenshot_files_alert_added.png)
+
+### Additional Alert on Videos Page
+![🍞 in breadcrumbs slot](./screenshot_videos_alert_added.png)
```

Comment:
Why is there a second screenshot of the videos page if this slot is for the files page?

Reply:
Oh, wouldn't it be more common to want to display different things on those pages? If this forces you to display the same alert on both, that seems overly specific to one use case. I'm wondering if we should split this into two slots.